openstack/sharedfilesystems/v2/securityservices/requests.go

```diff
@@ -0,0 +1,52 @@
+package securityservices
+
+import "github.com/gophercloud/gophercloud"
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToSecurityServiceCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains options for creating a SecurityService. This object is
+// passed to the securityservices.Create function. For more information about
+// these parameters, see the SecurityService object.
+type CreateOpts struct {
+	// The security service name
+	Name string `json:"name"`
+	// The security service description
+	Description string `json:"description,omitempty"`
+	// The security service type. A valid value is ldap, kerberos, or active_directory
```

Comment:
Are these the only [3] types allowed? If so, we should probably create a type for this that has those three as `const`s of that type.